pkgs/by-name/cl/cloud-ddns/package.nix

```
@@ -0,0 +1,35 @@
+{
+  lib,
+  buildGoModule,
+  fetchFromGitHub,
+}:
+
+buildGoModule rec {
+  pname = "cloud-ddns";
+  version = "0.5";
+
+  src = fetchFromGitHub {
+    owner = "jason-m";
+    repo = pname;
+    rev = "70b82c9";
```

You have tags, use them
```suggestion
    tag = "v${finalAttrs.version}";
```
(also do not use shortened rev hashes, in large repos, like Nixpkgs and the kernel, they become very ambiguous)